doc/nrf/known_issues.rst

```diff
@@ -1886,6 +1886,23 @@ Secure Partition Manager (SPM)
 .. note::
     The Secure Partition Manager (SPM) is deprecated as of |NCS| v2.1.0 and removed after |NCS| v2.2.0. It is replaced by :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
 
+.. rst-class:: v2-2-0
+
+NCSDK-19156: Building SPM for other boards than ``nrf5340dk_nrf5340_cpuapp`` and ``nrf9160dk_nrf9160`` fails with compilation error in cortex_m_systick.c
+  This is because the :kconfig:option:`CONFIG_CORTEX_M_SYSTICK` is enabled while the systick node is disabled in devicetree.
+
+  **Workaround** Enable systick in a DTS overlay file for the SPM build.
+
+  #. Create an overlay file systick_enabled.overlay that enable systick node.
+
+    .. code-block::
```

Comment:
Make sure to double-check the alignments.
It looks like the code block is not properly aligned, and you can see it in the doc build.